python/hsfs/core/vector_server.py

```diff
@@ -373,7 +378,7 @@ def get_inference_helper(self, entry, return_type):
         elif return_type.lower() == "polars":
             # Polar considers one dimensional list as a single columns so passed vector as 2d list
             polars_df = pl.DataFrame(
-                [serving_vector], schema=self._feature_vector_col_name, orient="row"
+                [serving_vector], schema=self._inference_helper_col_name, orient="row"
```

Comment:
Changed it to use serving_vector.keys() for `get_inference_helper`.

For `get_inference_helpers` since `batch_results` is a list I used `feature_view_object.inference_helper_columns` which was a pre-existing variable in feature view. I am wondering if I should use it for  `get_inference_helper` too?